demos/mqtt/mqtt_demo_basic_tls/mqtt_demo_basic_tls.c

```diff
@@ -235,11 +235,14 @@ static int connectToServerWithBackoffRetries( NetworkContext_t * pNetworkContext
  * MQTT_PUBLISH_COUNT_PER_LOOP number of times, and verifies if it
  * receives the Publish message back.
  *
- * @param[in] pNetworkContext Pointer to the network context created using Openssl_Connect.
+ * @param[in] pMqttContext MQTT context pointer.
+ * @param[in, out] pClientSessionPresent Pointer to flag indicating if an
```

Comment:
Is this OK for doxygen, or does it need to be `@param[in,out]`, with no spaces?

Reply:
Couldn't find any specific doc about it. I will remove the space to be consistent. Thanks.

Reply:
Why is this marked as resolved?